torch/nn/parallel/distributed.py

```diff
@@ -947,6 +957,16 @@ def no_sync(self):
         finally:
             self.require_backward_grad_sync = old_require_backward_grad_sync
 
+    def _run_ddp_forward(self, *inputs, **kwargs):
+        from .replicated_tensor_ddp_interop import _ddp_with_replicated_tensor_enabled
```

Comment:
Do we need to do this import on every forward pass? Will it affect performance?

Reply:
yeah seems like we can just import it once in the header?

Reply:
Good point, will move this up.